test/src/commands/create/create_test.dart

```diff
@@ -217,6 +220,7 @@ void main() {
             'project_name': 'my_app',
             'org_name': 'com.example.verygoodcore',
             'description': '',
+            'executable_name': 'my_app',
```

Comment:
Is not required for this tests right?

Reply:
It is, otherwise, the verification doesn't match.